src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+
+function sortList(list) {
+  return list.sort((a, b) => (
+    b.dataset.salary.replace(/\D/g, '') - a.dataset.salary.replace(/\D/g, '')));
```

Comment:
```suggestion
    b.dataset.salary.replace(/\D/g, '') - a.dataset.salary.replace(/\D/g, '')));
```

according to task, you should create function for converting salary string into number